src/graphql/root/query/globals.ts

```diff
@@ -22,6 +24,8 @@ const GlobalsQuery = GT.Field({
       lightningAddressDomain: getLightningAddressDomain(),
       lightningAddressDomainAliases: getLightningAddressDomainAliases(),
       buildInformation: getGaloyBuildInformation(),
+      smsAuthSupportedCountries: getSmsAuthSupportedCountries(),
+      whatsAppAuthSupportedCountries: getWhatsAppAuthSupportedCountries(),
```

Comment:
didnt see this before my last comment but.... if you create a Country type (or use the name that you prefer) we will be able to include the supported methods and will only need 1 query

Reply:
I can switch to that schema. Do you think it needs to be in the domain or is it okay just being a type in gql layer?